web/source/keyboards/kmwkeyboards.ts

```diff
@@ -121,9 +125,9 @@ namespace com.keyman.keyboards {
 
     firstCall: boolean = true; // First time to call keymanCloudRequest()
 
-    deferredStubs: any[] = []; // The list of user-provided keyboard stub registration objects.
-    deferredKRS = [];          // Array of pending keyboard stubs from KRS, to register after initialization
-    deferredKR = [];           // Array of pending keyboards, to be installed at end of initialization
+    // For deferment of adding keyboards until keymanweb initialized
```

Comment:
```suggestion
    // For deferment of adding keyboards until keymanweb initializes
```

Verb tense conflict.